src/components/NavSection.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="Home">
+    <sidebar-section v-if="showSidebar" />
+    <nav>
+      <div class="logo">
+        <img src="@/assets/images/Icons/Group 2.svg" alt="" id="logo" />
+      </div>
+      <div class="nav-options">
+        <div class="nav-icon">
+          <img src="@/assets/images/Icons/Rectangle.svg" alt="" id="nav-icon" />
+        </div>
+        <ul>
+          <li> <router-link to ="/"><span>00</span>HOME</router-link></li>
+          <li><span>01</span>DESTINATION</li>
+          <li><span>02</span>CREW</li>
+          <li><span>03</span>TECHNOLOGY</li>
+        </ul>
+      </div>
+      <div class="nav-icons" @click="toggleIcon">
+        <img
+          src="@/assets/images/Icons/Group (2).svg"
+          alt=""
+          id="hamburger-icon"
+          v-if="showIcon"
```

Comment:
Change to `v-if="!showSidebar"`, so you can get rid of `showIcon`
You don't need to have both